lte/gateway/fabfile.py

```diff
@@ -454,15 +454,16 @@ def integ_test_containerized(
 
 def _start_gateway_containerized(c_gw, docker_registry=None):
     """ Starts the containerized AGW """
-    with c_gw.cd(AGW_PYTHON_ROOT):
-        c_gw.run('make buildenv')
 
-    with c_gw.cd(AGW_ROOT):
-        c_gw.run(
-            'for component in redis nghttpx td-agent-bit; do cp "${MAGMA_ROOT}"'
-            '/{orc8r,lte}/gateway/configs/templates/${component}.conf.template;'
-            ' done',
-        )
+    c_gw.run('sudo rm -rf /etc/snowflake && sudo touch /etc/snowflake')
+    c_gw.run('cd ${MAGMA_ROOT} && bazel/scripts/link_scripts_for_bazel_integ_tests.sh')
+    c_gw.run('cd ${MAGMA_ROOT} && bazel build `bazel query "attr(tags, util_script, kind(.*_binary,//orc8r/... union //lte/...))"`')
```

Comment:
Nit: you could put these two lines in a `with c_gw.cd("${MAGMA_ROOT}")` context instead of manually using `cd` in each run command.

Reply:
Done